src/metalnx-web/src/main/javascript/gallery/components/Gallery.vue

```diff
@@ -1,8 +1,20 @@
 <template>
   <!-- Search -->
-  <div>
+  <div v-if="!thumbnails">
+    Error occurs when fetching data from iRODS Rule Engine.
+  </div>
+  <div v-else-if="thumbnails.error">iRODS Rule Engine does not exist.</div>
```

Comment:
'An error has occurred while fetching thumbnail data from iRODS.'

?   can we be more specific... all we know is that it didn't come back good.  right?